client/my-sites/plan-features-2023-grid/components/actions.tsx

```diff
@@ -109,16 +133,31 @@ const LaunchPagePlanFeatureActionButton = ( {
 		);
 	}
 
+	let buttonText;
+
+	if ( isStuck ) {
```

Comment:
`Non-blocking`:

Wondering how we feel about using the term isStickied instead?